Common/include/option_structure.hpp

```diff
@@ -895,6 +895,9 @@ static const MapType<std::string, ENUM_MATRIX_COLORING> MatrixColoring_Map = {
 enum class LIMITER {
   NONE                 , /*!< \brief No limiter. */
   VENKATAKRISHNAN      , /*!< \brief Slope limiter using Venkatakrisnan method (stencil formulation). */
+  NISHIKAWAR3          , /*!< \brief Slope limiter using NISHIKAWAR3 method (stencil formulation). */
```

Comment:
For consistency with other names please separate with _
```suggestion
  NISHIKAWA_R3          , /*!< \brief Slope limiter using Nishikawa's R3 method (stencil formulation). */
```